File: Java/bubble.java
import java.util.Arrays;

public class Main {
    public static void main(String[] args) {
        int[] x = {3, 2, 1, 4, 5};
        System.out.println(Arrays.toString(x));
        bubble(x);
        System.out.println(Arrays.toString(x));
    }

    private static void bubble(int[] x) {
        int n = x.length;
        for (int i = 0; i < n - 1; i++)
            for (int j = 0; j < n - i - 1; j++)
                if (x[j] > x[j + 1]) {
                    int temp = x[j];
                    x[j] = x[j + 1];
                    x[j + 1] = temp;
                }
    }
}
// (c) Valentin Arkov

File: Python/bubble.py
def bubble(x):
    n = len(x)
    for i in range(n - 1):
        for j in range(0, n - i - 1):
            if x[j] > x[j+1]:
                x[j], x[j+1] = x[j+1], x[j]
    return x

x = [3, 2, 1, 4, 5]
print(x)
print(bubble(x))

# (c) Valentin Arkov
